test/gpu/amdgpu.jl

```diff
@@ -6,10 +6,18 @@ using LinearOperators, AMDGPU
   B = ROCArray(rand(Float32, 10, 10))
   C = ROCArray(rand(Float32, 20, 20))
   M = BlockDiagonalOperator(A, B, C)
+  v = ROCArray(rand(5))
+
 
   v = ROCArray(rand(Float32, 35))
```

Comment:
declaring v as `ROCArray(rand(5))` here is immediately overwritten by the next assignment to `ROCArray(rand(Float32, 35))`.